### DISCUSSION TITLE
Adding a second lightspot, generates a bug

### DISCUSSION THREAD

**Discussion by NoelKi** [pyvista/pyvista#3197] [Q&A]
Hey guys, 
I tryed to bring shadows to my spherical surface, which worked really well for one spotlight. See the cyan light. But wen I try to implement another spotlight like the brown one you can see some spaces which aren't lighted at all. Does anyone now how to fix this ? 

btw. I tryed to bring the angle of the first light above 90°, which creates a lightspot in all directions, this works really well. But then it is impossible to create shadows, thats why i tryed to create multiple lightspots. 

Thanks for any help an sorry for typing mistakes I'm not  a native speaker. 
![2nd light problem](https://user-images.githubusercontent.com/111748573/185915326-77326f05-98fc-44b5-943d-5d5b1f0c80a8.PNG)

**Comment by akaszynski** [comment]
Hello,

Could you please post the code you used to generate this?
